definitions/scenarios/satellite_upgrade.rb

```diff
@@ -23,14 +23,43 @@ class PreUpgradeCheck < Abstract
       run_strategy :fail_slow
     end
 
+    # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
```

Comment:
What's this for?

Reply:
This method is just too long so we disable them just for this